tools/build_chain.sh

```diff
@@ -1286,6 +1294,66 @@ check_bin()
     echo "Binary check passed."
 }
 
+prepare_ca(){
+
+    if [ -z ${CertConfig} ] || [ ! -e ${CertConfig} ];then
+        # CertConfig="${output_dir}/cert.cnf"
+        generate_cert_conf "cert.cnf"
+    else 
+        cp ${CertConfig} .
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5331982)